.github/workflows/operator.yml

```diff
@@ -0,0 +1,165 @@
+name: Operator Release
```

Comment:
Please merge back the master to include the pipeline changes as well otherwise, we could face some problems when releasing